edx_proctoring/templates/proctored_exam/ready_to_submit.html

```diff
@@ -28,6 +27,32 @@ <h3>
     </button>
   {% endif %}
 </div>
+<div id="accessible-error-modal" class="modal" aria-hidden="true">
```

Comment:
How about moving these modals to the footer.html, defining it only once and filling in the error message dynamically?

Reply:
Surprising detail: ready_to_submit doesn't use that footer. Looking at the commit history I'm not confident that discrepancy is unintentional; there's copy in the footer with which we may not want to distract the user.

Reply:
Now this same modal is in three templates. Surely there's a way to refactor this into one place.

Reply:
please don't call me shirley